### ISSUE TITLE
Categorical data

### ISSUE THREAD

**Issue by akablar** [scikit-learn-contrib/metric-learn#293]
Can this algorithm be used with a mix of numeric and categorical features? 

Are there any recommended data pre-processing/encoding methods?

**Comment by perimosocordiae** [comment]
None of our methods support categorical features directly, but we do have compatibility with the greater scikit-learn ecosystem which supports various feature preprocessing methods: https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing
